platform/provider.go

```diff
@@ -90,6 +90,8 @@ func NewProvider(logger boshlog.Logger, dirProvider boshdirs.Provider, statsColl
 		devicePathResolver = devicepathresolver.NewVirtioDevicePathResolver(idDevicePathResolver, mappedDevicePathResolver, logger)
 	case "scsi":
 		devicePathResolver = devicepathresolver.NewScsiDevicePathResolver(500*time.Millisecond, fs)
+	case "scsiid":
+		devicePathResolver = devicepathresolver.NewScsiIDDevicePathResolver(50000*time.Millisecond, fs, logger)
```

Comment:
There are some activities of hypervisor and guest OS. I am pretty sure it is related to slow hardware or busy system because I can't repo it in our test environment. We are gathering more logs to understanding what activities take this long.